content/en/platform/corda/5.0/introduction/use-cases.md

```diff
@@ -8,4 +8,20 @@ menu:
     weight: 2000
 section_menu: corda5
 ---
-"Who is Corda for?"
\ No newline at end of file
+"_Who is Corda for?_" Corda was originally designed as a distributed ledger and system of record for the financial services industry. 
+Within financial markets, Corda can be used to automate trading and settlement processes and reduce the need for reconciliation. 
+Additionally, Corda’s smart contracts make it easy to automate complex financial agreements between parties. Corda is used in financial markets for use cases such as:
+
+* **Financial Market Infrastructure (FMI)** - Corda accelerates and automates settlement processes, thereby removing the need for costly and risky reconciliation processes, reducing credit, and counter-party risk and default risk arising from unsettled trades.
+* **Central Banks** - Corda enables dematerialization (also referred to as tokenization or digitalization) of central bank money into central bank digital currency (CBDC). This is applied to the replacement of RTGS (real-time gross settlement) systems, cross-border payments, and delivery versus payment (DvP) scenarios. 
+* **Banks** - Corda facilitates the overnight reconciliation of inter-bank transactions and financial asset management, including issuance, trading, and custody of dematerialized financial assets such as bonds and equities. Also, it facilitates the issuing of private money (as fiat backed digital currency) for inter-book cross-border transactions and delivery versus payment (DvP) scenarios.
+* **Exchanges** - Corda manages the lifecycle of digital financial assets and automating processes such as issuance, trading, settlement and custody.
+* **FinTechs** - Corda enables the building of solutions for the digitalizing of traditional assets, such as bill of lading, letters of credit, and precious commodities in order to increase efficiencies and reduce business risk.
+ 
+Corda’s capabilities are also applicable across a broader set of markets and industries where there is a need for multi-party workflows managing transactions, data, and processes to establish a shared view of a distributed set of data, i.e. I see what you see. These include industries such as:
+
+* **Insurance** - Insurance companies use Corda to automate the settlement of claims and other transactions. Corda's smart contracts make it easy to enforce the terms of insurance policies and automate the claims process.
+* **Supply chain management** Corda can be used in supply chain management to track goods and ensure that they are authentic and have not been tampered with. Corda's privacy features make it well-suited for managing confidential information in supply chain transactions.
+* **Healthcare** Due to Corda being strong on privacy and confidentiality, it can be used in the healthcare industry to manage and provide access to patient data securely and ensure that patient privacy is protected.
```

Comment:
```suggestion
* **Healthcare** - Due to Corda being strong on privacy and confidentiality, it can be used in the healthcare industry to manage and provide access to patient data securely and ensure that patient privacy is protected.
```